tests/client/amo/components/TestAddonDetail.js

```diff
@@ -324,4 +325,12 @@ describe('AddonDetail', () => {
 
     assert.ok(rootNode.querySelector('.AddonMoreInfo-contents'));
   });
+
+  it('renders meta data', () => {
+    const root = render({ addon: fakeAddon });
```

Comment:
Mind testing the user count directly? Either check that it's `100` since that's the value in `fakeAddon` or explicitly define it (I prefer the latter).

```js
const addon = { ...fakeAddon, average_daily_users: 25 };
const root = render({ addon });
// ...
assert.equal(metaData.props.averageDailyUsers, 25);